meta-lmp-base/recipes-support/resize-helper/resize-helper/resize-helper

```diff
@@ -78,4 +86,11 @@ if [ -n "${PART_ENTRY_NAME}" ]; then
 fi
 
 ${PARTX} -u ${DEVICE}
-${RESIZE2FS} "${ROOT_DEVICE}"
```

Comment:
`ROOT_DEVICE` is handled if on device-mapper in line 49 so what we need is only call the `cryptsetup resize` before calling the `resize2fs`

```
if [ -n "${DM_NAME}" ] && cryptsetup isLuks ${ROOT_DEVICE}; then
	# OP-TEE: use TEE Identity for pkcs11 authentication
	export CKTEEC_LOGIN_TYPE=user
	cryptsetup resize --token-only ${DM_NAME}
fi
${RESIZE2FS} "${ROOT_DEVICE}"

```

Reply:
So here we actually need a different resize2fs line because what we need to resize is the mounted crypt device (via DM NAME) and not the original block device given (which is the encrypted block device).

Reply:
right, please drop my comment.